### ISSUE TITLE
Umlaute werden falsch dargestellt

### ISSUE THREAD

**Issue by blackbasket** [homematic-community/hm_email#28]
Hallo zusammen,

ich habe seit dem Update auf RM 3.63.9.20220521 und e-mail Addon 1.7.3 Probleme mit den Umlauten.

Bis zum Update am Donnerstag lief alles problemlos. Das encoding habe ich gemäß Anleitung auf `convertfrom utf-8` umgestellt. Leider ohne Erfolg...

In der WebUI der RM wird der Text korrekt angezeigt.

LG,
Marcel

**Comment by dahmen** [comment]
Hallo Jens,
das Verhalten habe ich auch bemerkt.
MUA: Thunderbird
RPI-CCU: 3.63.9.20220521

Im Quellcode der Mail steht ein "ae, oe, ue" richtig als äöü.
Das Subject ist im Quellcode der Mail entcodet: "Subject: =?UTF-8?B?SG9tZU1hdGljIChGZWhsZXIpOiBTY3JpcHQ6IDEg?=
	=?UTF-8?B?TWFpbCBTZW5kIPbk/A==?="
sollte am Ende auch ein äöü enthalten.
Beide stellen, Subject und Body werden im Thunderbird aber falsch dargestellt.
![grafik](https://user-images.githubusercontent.com/19546516/174228324-102c406f-2e7d-4cd3-b6ca-29d720ae69d7.png)
![grafik](https://user-images.githubusercontent.com/19546516/174228377-9d15688e-6a68-4c15-803c-c761def24ec2.png)

VG
Christian

**Comment by Al-Bund** [comment]
Hallo zusammen.
Das Umlautproblem scheint nur behoben worden zu sein, wenn die Umlaute in der E-Mail Addon geschrieben wurden. Kommt der Text aus einem Skrip im Programm, werden die Umlaute falsch dargestellt.
Hier die Bilder des Tests dazu:
![Addon](https://user-images.githubusercontent.com/84993560/185500240-37a37734-1fde-4b7b-a835-28e44dd915bf.p
![E-Mail](https://user-images.githubusercontent.com/84993560/185500273-986317bc-139a-425d-ac2f-315ab454c710.png)

![Skript](https://user-images.githubusercontent.com/84993560/185500259-7c1dad26-6ae3-4e65-8c9c-cd08ddfb802f.png)
ng)
Systeminfo: RaspberryMatic v3.65.6.20220723 mit E-Mail Addon 1.7.4
Ist das bei euch auch noch so?

**Comment by dahmen** [comment]
i still have the same problem Jens.

**Comment by addy38** [comment]
I have the issue also in the actual version 1.7.6
`NEUE Servicemeldung
WZ Terrasse T�r :
Konfigurationsdaten stehen zum Transfer an
2 mal seit: 28.06.23 08:44 ( zuletzt: 28.06.23 08:44)`

**Comment by blackbasket** [comment]
Ja, bei mir leider auch weiterhin... wenn ich irgendwas zur Hilfe beitragen kann, gerne @jens-maus

**Comment by blackbasket** [comment]
ok, ich hab's jetzt wieder richtig... ich bin auf meine ganz alte TCL-config gegangen, in der ich das Gegenteil der Anleitung mache... 

also `set body [encoding convertto utf-8 $values(body)]`
anstatt `set body [encoding convertfrom utf-8 $values(body)]`

@addy38 hilft dir das auch?

LG,
Marcel

**Comment by addy38** [comment]
Hallo, ja mit`set serviceliste [encoding convertto utf-8 [lindex $argv 1]]` werden die Umlaute auch bei mir wieder richtig dargestellt
